scripts/TeamMemberVesting.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+// Scripting tool
+import {Script} from "../lib/forge-std/src/Script.sol";
+
+import {FlowVestor} from "../contracts/FlowVestor.sol";
+import "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+
+contract TeamMemberVesting is Script {
+    address private constant TEAM_MULTI_SIG = 0x13eeB8EdfF60BbCcB24Ec7Dd5668aa246525Dc51;
+    // TODO: reset DEPLOYER_EOA
+    address private constant DEPLOYER_EOA = 0xD93142ED5B85FcA4550153088750005759CE8318;
+    // team member addresses
+    address private constant T0RB1K = 0x0b776552c1Aef1Dc33005DD25AcDA22493b6615d;
+    address private constant MOTTO = 0x78e801136F77805239A7F533521A7a5570F572C8;
+    address private constant COOLIE = 0x03B88DacB7c21B54cEfEcC297D981E5b721A9dF1;
+
+    // TODO: Fill the address
+    address private constant FLOW = 0x78e489523291581205Ea3fA16a69689EcA79757A;
```

Comment:
@0xMotto can we add a revert in the run function for these scripts until FLOW address is confirmed? or do you think it's not needed?

Reply:
actually forget about it since mainnet deployment is happening very soon